activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -4,6 +4,22 @@
 
 module I18n
   class Railtie < Rails::Railtie
+    module Helpers
```

Comment:
I still think this should be moved to the path object. Can we find a new method that's stepping stone between the simpleness of `expanded_paths` and all the extra work `expanded` does. To me it seems there's code there to be shared - we just need to bring out the similarities more.